docs/_docs/installation/windows.md

```diff
@@ -97,7 +97,7 @@ Bash on Ubuntu on Windows is still under development, so you may run into issues
 
 ## Encoding
 
-If you use UTF-8 encoding, make sure that no `BOM` header characters exist in your files. If they don't, Jekyll will break. This is especially relevant when you're running Jekyll on Windows.
```

Comment:
I opted to shorten the paragraph to a single sentence by removing unnecessary words

Reply:
Layman users will not know what `BOM` represents. So asking them to *remove `BOM`* wouldn't exactly be user-friendly.
How does one remove something they don't see nor understand..?

Reply:
Thanks for the comment. I made a new commit